pkg/kapp/cmd/app/logs_flags.go

```diff
@@ -8,17 +8,19 @@ import (
 )
 
 type LogsFlags struct {
-	Follow       bool
-	Lines        int64
-	ContainerTag bool
-	PodName      string
+	Follow        bool
+	Lines         int64
+	ContainerName []string
+	ContainerTag  bool
+	PodName       string
 }
 
 func (s *LogsFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().BoolVarP(&s.Follow, "follow", "f", false, "As new pods are added, new pod logs will be printed")
 	cmd.Flags().Int64Var(&s.Lines, "lines", 10, "Limit to number of lines (use -1 to remove limit)")
 	cmd.Flags().BoolVar(&s.ContainerTag, "container-tag", true, "Include container tag")
 	cmd.Flags().StringVarP(&s.PodName, "pod-name", "m", "", "Set pod name to filter logs (% acts as wildcard, e.g. 'app%')")
+	cmd.Flags().StringSliceVarP(&s.ContainerName, "container-name", "c", nil, "Set container names to filter logs in the pod, separated by comma")
```

Comment:
oops, should have mentioned that actual field should stay plural (eg ContainerNames), only flag is singular.

Reply:
No problem :)